### DISCUSSION TITLE
3.5 UI: Filter out all archived WFs in "Workflows" page?

### DISCUSSION THREAD

**Discussion by ni-todo-spot** [argoproj/argo-workflows#13374] [Q&A] [answered]
Hey all!

Just upgraded our argo to the newest version and noticed there is no more archive page, all WFs are in one page with 1 field indicating whether it's cached or not.
This got me to think if can filter all archived WFs and view those which are not.

Thanks in advance!

**Comment by Joibel** [comment, marked as answer]
This is already a feature request here: https://github.com/argoproj/argo-workflows/issues/13171
